testing/examples/state_tests.rs

```diff
@@ -89,13 +89,23 @@ async fn main() {
 	let _keys = api.get_keys(storage_key, None).unwrap().unwrap();
 	let _constants: Balance = api.get_constant("Balances", "ExistentialDeposit").unwrap();
 
-	let max_keys = 3;
+	let max_keys = 2003;
+	let result =
+		api.get_storage_keys_paged(Some(storage_key_prefix.clone()), max_keys.clone(), None, None);
+	assert!(result.is_err());
+	assert!(format!("{result:?}").contains("count exceeds maximum value"));
+
 	let storage_keys = api
-		.get_storage_keys_paged(Some(storage_key_prefix), max_keys, None, None)
+		.get_all_storage_keys_paged_up_to_count(Some(storage_key_prefix), max_keys, None, None)
 		.unwrap();
-	assert_eq!(storage_keys.len() as u32, max_keys);
+	assert!(storage_keys.len() as u32 > 3);
+	assert!(storage_keys.len() as u32 <= max_keys);
```

Comment:
Any chance to assert that it is equal to `max_keys`? Only then we will know that our function is working correctly. 
Maybe change the prefix to `None` ?

Reply:
Good question.. I don't know 😨  I'd simply put a random number above 1000 and see what the assert statement prints